Source/WebCore/rendering/GridMasonryLayout.cpp

```diff
@@ -166,4 +199,14 @@ LayoutUnit GridMasonryLayout::offsetForChild(const RenderBox& child) const
     return offsetIter->value;
 }
 
+inline GridTrackSizingDirection GridMasonryLayout::gridAxisDirection() const
+{
+    return m_masonryAxisDirection == ForRows ? ForColumns : ForRows;
```

Comment:
Would like a comment here to mention that the masonry and grid axis are perpendicular.

Reply:
Done let me know if you want it changed at all

Reply:
Looks good.